rilldata/rill
# Persist some dashboard state to local storage and use it when returning to a dashboard

## Checklist

## Summary
#### Issue addressed: 
closes #3932

#### Details:
We are storing the selection of measures, dimensions, selected leaderboard measure and leaderboard sort settings.

## Steps to Verify
1. Create a few dashboards.
2. Navigate to them and change the settings mentioned in details sections.
3. Close the page and go back to those dashboards.
4. The above settings should be retained.
